src/components/molecules/Settings/Account/AccountSection/index.tsx

```diff
@@ -105,4 +115,19 @@ const StyledIcon = styled(Icon)`
   }
 `;
 
+const Notice = styled.div`
+  margin-left: 184px;
+  display: flex;
+`;
+
+const AlertIcon = styled(Icon)`
+  color: ${({ theme }) => theme.colors.dark.functional.attention};
+`;
+
+const StyledText = styled(Text)`
+  margin-left: 12px;
+  width: 500px;
```

Comment:
If you can move the width into the wrapping Flex(currently the Notice component) I think that is clearer and less prone to issues in the future.